app/src/main/java/it/quip/android/partials/EndlessScrollHandler.java

```diff
@@ -0,0 +1,47 @@
+package it.quip.android.partials;
```

Comment:
"Interfaces" shouldn't be a package name. The interfaces should lie within the appropriate packages. That would be like making a "classes" or "enums" package